app/layout.tsx

```diff
@@ -32,10 +20,10 @@ export default function RootLayout({
   children: React.ReactNode;
 }) {
   return (
-    <html lang="en">
+    <html lang="en" className={`${GeistSans.variable} ${GeistMono.variable}`}>
       <PHProvider>
         <body
-          className={`${inter.variable} ${architects_daughter.variable} font-inter bg-background tracking-tight text-gray-900 antialiased`}
+          className={`--font-geist-sans bg-background tracking-tight text-gray-900 antialiased`}
```

Comment:
This is wrong we should just use the tailwind class to apply the font
```suggestion
          className={`font-sans bg-background tracking-tight text-gray-900 antialiased`}
```